### DISCUSSION TITLE
Time of release of v2.0.5?

### DISCUSSION THREAD

**Discussion by Jason2866** [espressif/arduino-esp32#7146] [Q&A]
@VojtechBartoska Do you have already a date when you plan to release v2.0.5 ?

**Comment by VojtechBartoska** [comment]
Hi Jason, we would like to have it at max. on Friday 26.8.

It was planned to be earlier but we faced some important issues.

I will share sprint Meetings notes tomorrow so there will be more clear description. I was a bit busy and wasn't able to do it earlier.

**Comment by Jason2866** [comment]
Hello Vojtech,
thx for the fast answer. Great to hear that it is planned in August!
